GetARock/GetARock/Screens/Maps/MainMapViewController.swift

```diff
@@ -106,10 +120,74 @@ final class MainMapViewController: UIViewController {
         
     }
     
+    private func setLocationManager() {
+        self.locationManager.delegate = self
+        self.locationManager.desiredAccuracy = kCLLocationAccuracyBest
+        self.locationManager.requestWhenInUseAuthorization()
+    }
+    
+    private func moveLocation(to coordinate: CLLocationCoordinate2D?) {
+        guard let coordinate = coordinate else { return }
+        
+        self.currentCoordinate.latitude = coordinate.latitude
+        self.currentCoordinate.longitude = coordinate.longitude
+        
```

Comment:
p3;
```suggestion
        self.currentCoordinate = coordinate
```
coordinate 바로 넣어줘도 좋을 것 같아요!

Reply:
엥 그러게요 
이때는 일을 2번 하고 싶었나 봅니다... 머쓱..
확인 감사합니다! d61b2a28df0c09cc1612f80f3d79e4402b5022ce